src/App.js

```diff
@@ -1,36 +1,47 @@
 import React, { Component } from 'react';
-import movieData from './movieData';
-import Movies from './Movies';
+import { movieData } from './apiCalls.js';
+import Movies from './Movies.js';
 import './App.css';
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: movieData.movies,
+      movies: [],
     }
   }
 
+  componentDidMount() {
+    movieData()
+    .then(movies => this.setState({ movies: movies.movies }))
+    .catch(error => console.log(error))
+  }
+
   findPoster = (id) => {
     const selectedMovie = this.state.movies.filter(movie => movie.id === id);
 
     this.setState({ movies: selectedMovie });
   }
 
-  render() {
-    const movieInfo = 
+  setMovieInfo = () => {
+    return(
       <section className='movie-info'>
         <p>{this.state.movies[0].title}</p>
         <p>Average Rating: {this.state.movies[0].average_rating}</p>
         <p>Release Date: {this.state.movies[0].release_date}</p>
         <button onClick={() => window.location.reload()}>Go Back</button>
       </section>
-    
+    )
+  }
+
+
+
+  render() {
     return (
       <section>
         <h1 className='page-title'>Rancid Tomatillos</h1>
         <Movies movies={this.state.movies} findPoster={this.findPoster}/>
-        {this.state.movies.length === 1 && movieInfo}
+        {this.state.movies.length === 1 &&  this.setMovieInfo()}
```

Comment:
Breaking this out into a helper function makes sense.  We don't want to be rendering unnecessary code.

Reply:
I actually think that because this helper function is only returning JSX, the logic could live inside the render. The naming of the helper function I think makes this less readable - setMovieInfo doesn't tell me clearly what is being rendered.